#Three parallel vectors with the first containing the values and the second and third containing the x and y coordinates of the matching value
Header header
float32 resolution
uint32 width
uint32 height
float32 minValue
float32 maxValue
float32[] h_val
uint32[] h_x
uint32[] h_y
